src/components/themed/WalletListHeader.tsx

```diff
@@ -65,7 +66,7 @@ export class WalletListHeaderComponent extends React.PureComponent<Props> {
 
     return (
       <>
-        <View style={styles.searchContainer}>
+        <Space sideways around={0.5}>
```

Comment:
Why is this not using spaceProps?

Reply:
Technically, the Space component is using SpaceProps (that's all it uses). But, I think you're asking, why are we using a Space component here instead of putting the space prop in the SimpleTextInput. This is because we need a container component to layout the child elements in a row and the `sideways` prop acheives that.